beets/util/__init__.py

```diff
@@ -949,17 +949,21 @@ def interactive_open(targets, command):
     """
     assert command
 
-    # Split the command string into its arguments.
-    try:
-        args = shlex.split(command)
-    except ValueError:  # Malformed shell tokens.
+    if os.name == 'nt':
         args = [command]
+    else:
+        # Split the command string into its arguments.
+        try:
+            # Doesn't work well with Windows (removes all the '\' from path resulting in an invalid path)
+            args = shlex.split(command)
+        except ValueError:  # Malformed shell tokens.
+            args = [command]
 
-    args.insert(0, args[0])  # for argv[0]
+    # Unnecessary for subprocess.call()
+    # args.insert(0, args[0])  # for argv[0]
 
     args += targets
-
-    return os.execlp(*args)
```

Comment:
Well, there is a reason we're using `execlp`… it "replaces" the beets process with the new editor process, which is a bit nicer on people's systems. There isn't a useless Python process just taking up memory, waiting for the editor to close.